cmd/root.go

```diff
@@ -51,6 +51,16 @@ func Execute() {
 	}
 }
 
+func RunPreRoot(ro *options.RootOptions) error {
```

Comment:
what's going on here?  it seems to be a copy paste of the function below and isn't used.

Reply:
I need an exported function for Scribe

Reply:
Scribe should either set up it's own logging and copy paste this to Scribe internally, or rename the existing function to export it instead of copy/pasting it.  If we are exporting these functions they should be named something more specific than `RunPreRoot`, something like `SetupLogging`.